android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -181,9 +183,35 @@ class AddonsManagerAdapter(
         }
     }
 
-    override fun onBindViewHolder(holder: CustomViewHolder, position: Int) {
+    override fun onBindViewHolder(holder: CustomViewHolder, @SuppressLint("RecyclerView") position: Int) {
         val item = getItem(position)
 
+        // Configure an accessibility delegate for each item.
+        holder.itemView.accessibilityDelegate = object : View.AccessibilityDelegate() {
+            override fun onInitializeAccessibilityNodeInfo(host: View, info: AccessibilityNodeInfo) {
+                super.onInitializeAccessibilityNodeInfo(host, info)
+
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
+                    info.collectionItemInfo = AccessibilityNodeInfo.CollectionItemInfo(
+                        position,
```

Comment:
I was taking a look at the`@SuppressLint("RecyclerView")` , it seems that we shouldn't be using the `position` parameter, instead we should be using `getAdapterPosition()` , bellow the reference:


> Do not treat position as fixed; only use immediately and call holder.getAdapterPosition() to look it up later More... (⌘F1)  Inspection info: RecyclerView will not call onBindViewHolder again when the position of the item changes in the data set unless the item itself is invalidated or the new position cannot be determined.  For this reason, you should only use the position parameter while acquiring the related data item inside this method, and should not keep a copy of it.  If you need the position of an item later on (e.g. in a click listener), use getAdapterPosition() which will have the updated adapter position.  Issue id: RecyclerView Vendor: Android Open Source Project Contact: https://groups.google.com/g/lint-dev Feedback: https://issuetracker.google.com/issues/new?component=192708

Reply:
We don't do this in the codebase elsewhere so I dunno

Reply:
updated